beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperDryRunMetadataCleanupIntegrationTest.java

```diff
@@ -252,16 +253,21 @@ public void dryRunDontDropPartitionedTable() throws Exception {
   }
 
   @Test
-  public void dryRunMetrics() throws SQLException, TException {
-    hiveTestUtils.createTable(UNPARTITIONED_TABLE_PATH, TABLE_NAME_VALUE, false);
-    amazonS3.putObject(BUCKET, UNPARTITIONED_TABLE_OBJECT_KEY, "");
+  public void dryRunMetrics() throws Exception {
+    Table table = hiveTestUtils.createTable(PARTITIONED_TABLE_PATH, TABLE_NAME_VALUE, true);
```

Comment:
Why has the table become partitioned in this test?

Reply:
I changed it so I can check all of the tags are used. For an unpartitioned table it wouldnt use the tag `hive-partition-deleted`, so instead of adding another test for that I changed the table type.

Reply:
I see. Would it be possible to just add it instead (e.g. `dryRunMetricsPartitioned`), so the unpartitioned table still gets tested? Or would it be a waste?

Reply:
can do but it would be testing the same tags - 1, so theres not much point imo. The table still gets tested, this is just specifically for the metrics.

Reply:
Ok then, we can leave it this way.